protoc-gen-twirp_ruby/main.go

```diff
@@ -70,20 +70,26 @@ func (g *generator) Generate(in *plugin.CodeGeneratorRequest) *plugin.CodeGenera
 func (g *generator) generateFile(file *descriptor.FileDescriptorProto) *plugin.CodeGeneratorResponse_File {
 	pkgName := pkgName(file)
 	g.P(`# Code generated by protoc-gen-twirp_ruby, DO NOT EDIT.`)
-	for _, service := range file.Service {
+	g.P(`require 'twirp'`)
+	g.P(``)
+	g.P(fmt.Sprintf("module %s", CamelCase(pkgName)))
```

Comment:
This will have to be a little more sophisticated to handle no-module (if no package), or multiple nested-modules (if package contains dots). But that can be done in the next PR